src/panels/developer-tools/service/developer-tools-service.js

```diff
@@ -276,7 +291,13 @@ class HaPanelDevService extends PolymerElement {
   _callService() {
     if (this.parsedJSON === ERROR_SENTINEL) {
       // eslint-disable-next-line
-      alert(`Error parsing YAML: ${this.serviceData}`);
+      alert(
+        this.hass.localize(
+          "ui.panel.developer-tools.tabs.services.alert_parsing_yaml",
+          "error",
```

Comment:
It is not the error but the data we can't parse to JSON.
```suggestion
          "data",
```

Reply:
TIL 😃 will fix it immediately